eppic-cli/src/main/java/eppic/assembly/CrystalAssemblies.java

```diff
@@ -587,19 +587,36 @@ public void score() {
 			indx++;
 		}
 		
+		// This is done when multiple assemblies are high scoring
+		indx = 0;
+		List<Integer> indices = new ArrayList<Integer>();
+		for (Assembly a:uniques) {
+			if (a.getScore() == maxScore){
```

Comment:
I'd say we should do this for strict ties only (those comparing as equals), let's trust the tiny differences to do the sorting job otherwise.